tools/mo/unit_tests/mo/front/tf/convert_to_pb_test.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import tempfile
+import unittest
+
+from openvino.tools.mo.front.tf.loader import convert_to_pb
+
+
+class ConvertToPBTests(unittest.TestCase):
+    test_directory = os.path.dirname(os.path.realpath(__file__))
+
+    def setUp(self):
+        self.argv = argparse.Namespace(input_model=None, input_model_is_text=False, input_checkpoint=None, output=None,
+                                       saved_model_dir=None, input_meta_graph=None, saved_model_tags=None,
+                                       model_name='model', output_dir=None)
+
+    def test_saved_model(self):
+        import tensorflow as tf
+        with tempfile.TemporaryDirectory(dir=self.test_directory) as tmp_dir:
+            inputs = tf.keras.Input(shape=(3,))
+            x = tf.keras.layers.Dense(4, activation=tf.nn.relu)(inputs)
+            outputs = tf.keras.layers.Dense(5, activation=tf.nn.softmax)(x)
+            model = tf.keras.Model(inputs=inputs, outputs=outputs)
+            model.save(tmp_dir)
+            self.argv.saved_model_dir = tmp_dir
+            self.argv.output_dir = tmp_dir
+            convert_to_pb(self.argv)
```

Comment:
What exactly is tested here? That convert_to_pb does not throw exceptions? Probably we could at least check if .pb file exists?